src/engine/test/scripts/engine-suite/src/engine_decoder/cmds/syntax_update.py

```diff
@@ -0,0 +1,62 @@
+import os, fnmatch
```

Comment:
Use pathlib and its walker instead of os, fnmatch

Reply:
ok